src/Cli/Utils.cs

```diff
@@ -898,6 +900,33 @@ public static bool WriteJsonContentToFile(string file, string jsonContent)
             return true;
         }
 
+        /// <summary>
+        /// This method will check if DAB_ENVIRONMENT value is set.
+        /// If yes, it will try to merge dab-config.json with dab-config.{DAB_ENVIRONMENT}.json
+        /// </summary>
+        public static string MergeConfigsIfAvailable()
+        {
+            string? environmentValue = Environment.GetEnvironmentVariable(RUNTIME_ENVIRONMENT_VAR_NAME);
+            if (!string.IsNullOrEmpty(environmentValue))
+            {
+                string FileToBeOverridden = DefaultName;
+                string FileToBeOverriddenWith = $"{CONFIGFILE_NAME}.{environmentValue}{CONFIG_EXTENSION}";
```

Comment:
The config name given an environment value is already given by the function RuntimeConfigPath.GetFileName

Can we reuse that function - so there is a single location to determine what the config name format should be in presence of environment variables?